src/main/java/Main.java

```diff
@@ -0,0 +1,18 @@
+import utils.WordsProcessingUtils;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        System.out.println("Please, provide a file name...");
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        String fileName = reader.readLine();
+        String[] result = WordsProcessingUtils.findLongestConcatenatedWords(fileName);
```

Comment:
```suggestion
        SomeClassThatRepresentTheResult  data = WordsProcessingUtils.findLongestConcatenatedWords(fileName);
```
повертати масив не зовсім круто. Не зрозуміло що означає result[0] а що означає result[2] і яка між ними різниц. Рекомендую створити окремий клас в якому будуть описані результати роботи алгоритму